build-tools/biome.json

```diff
@@ -0,0 +1,58 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.5.3/schema.json",
+	"files": {
+		"ignoreUnknown": true,
+		"ignore": ["**/_api-extractor-temp/**", "**/oclif.manifest.json", "**/_package.json"]
+	},
+	"vcs": {
+		"enabled": true,
+		"clientKind": "git",
+		"useIgnoreFile": true
+	},
+	"organizeImports": {
+		"enabled": false
```

Comment:
Have you tested enabling this? Curious how well it works. Definitely worth enabling in a separate PR if it does, but I would love to get auto import sorting enabled.

Reply:
I have tested it and it is quite good. Left for a separate PR. Details on implementation: https://biomejs.dev/analyzer/#how-imports-are-sorted

Reply:
Love it! So exciting!